src/Grid/Adaptor/Adaptor.component.tsx

```diff
@@ -27,144 +33,117 @@ interface FixedAdaptor extends CustomAdaptorProps {
+    const { max } = getMinMax(0, breakpoints);
+
+    return max !== 0 ? (
+        <Responsive className="anchor-adaptor" {...props} maxWidth={max} />
+    ) : null;
 };
 
-export const XS = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        maxWidth={StandardBreakpoints.xs.max}
-    />
-);
-export const SM = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        minWidth={StandardBreakpoints.sm.min}
-        maxWidth={StandardBreakpoints.sm.max}
-    />
-);
-export const MD = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        minWidth={StandardBreakpoints.md.min}
-        maxWidth={StandardBreakpoints.md.max}
-    />
-);
-export const LG = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        minWidth={StandardBreakpoints.lg.min}
-        maxWidth={StandardBreakpoints.lg.max}
-    />
-);
-export const XL = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        minWidth={StandardBreakpoints.xl.min}
-        maxWidth={StandardBreakpoints.xl.max}
-    />
-);
-export const XXL = ({ ...props }: FixedAdaptor) => (
-    <Responsive
-        className="anchor-adaptor"
-        {...props}
-        minWidth={StandardBreakpoints.xxl.min}
-    />
-);
-
-export const CenteredCell = styled('div')`
-    display: flex;
-    align-items: center;
-    justify-content: center;
-`;
-/* =========================================== STANDARD END ========================================================= */
+export const SM = ({ ...props }: FixedAdaptor) => {
+    const { breakpoints } = React.useContext(ResponsiveContext);
+    const { min, max } = getMinMax(1, breakpoints);
+
+    return min !== 0 && max !== 0 ? (
+        <Responsive
+            className="anchor-adaptor"
+            {...props}
+            minWidth={min}
+            maxWidth={max}
+        />
+    ) : null;
+};
+
+export const MD = ({ ...props }: FixedAdaptor) => {
+    const { breakpoints } = React.useContext(ResponsiveContext);
+    const { min, max } = getMinMax(2, breakpoints);
+
+    return min !== 0 && max !== 0 ? (
+        <Responsive
+            className="anchor-adaptor"
+            {...props}
+            minWidth={min}
+            maxWidth={max}
+        />
+    ) : null;
+};
+
+export const LG = ({ ...props }: FixedAdaptor) => {
+    const { breakpoints } = React.useContext(ResponsiveContext);
+    const { min, max } = getMinMax(3, breakpoints);
+
+    return min !== 0 && max !== 0 ? (
+        <Responsive
+            className="anchor-adaptor"
+            {...props}
+            minWidth={min}
+            maxWidth={max}
+        />
+    ) : null;
+};
+
+export const XL = ({ ...props }: FixedAdaptor) => {
```

Comment:
I don't know if these single breakpoint components are that useful, TBH. I'd like to explore making these better, but this re-solves what was here...